3MI0800270/2023-02-28/zad2.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+using namespace std;
+
+struct Tuple {
+    int first;
+    int second;
+
+    void input() {
+        cin >> first;
+        cin >> second;
+    };
+};
+
+int compareTuples(const Tuple& a, const Tuple& b) {
+    if (a.first > b.first || (a.first == b.first && a.second > b.second)) {
+        return 1;
+    }
+    else if (a.first == b.first && a.second == b.second) {
+        return 0;
+    }
+    else {
+        return -1;
+    }
+}
+int biggestTuple(const Tuple tuples[100]) {
+    int index = 0;
+    Tuple currBig = tuples[0];
+    for (int i = 1;i < 100;i++) {
+        if (compareTuples(tuples[i], currBig) == 1)
```

Comment:
Typically compare returns `negative`, `zero` or `positive` number. Checking for `1` is not a very good idea :P